auslib/test/admin/views/test_rules.py

```diff
@@ -859,20 +880,24 @@ def testUpdateScheduledChange(self):
         self.assertEquals(len(r), 1)
         db_data = dict(r[0])
         expected = {
-            "sc_id": 1, "when": 2000, "scheduled_by": "bill", "data_version": 2, "complete": False, "base_rule_id": 1, "base_priority": 100,
+            "sc_id": 1, "when": 2000000, "scheduled_by": "bill", "data_version": 2, "complete": False, "base_rule_id": 1, "base_priority": 100,
             "base_version": "3.5", "base_buildTarget": "d", "base_backgroundRate": 100, "base_mapping": "c", "base_update_type": "minor",
             "base_data_version": 1, "base_alias": None, "base_product": None, "base_channel": None, "base_buildID": None, "base_locale": None,
             "base_osVersion": None, "base_distribution": None, "base_distVersion": None, "base_headerArchitecture": None, "base_comment": None,
             "base_whitelist": None, "base_systemCapabilities": None, "telemetry_product": None, "telemetry_channel": None, "telemetry_uptake": None,
         }
         self.assertEquals(db_data, expected)
 
-    def testUpdateScheduledChangeCantRemoveProductWithoutPermission(self):
+    @mock.patch("time.time")
+    def testUpdateScheduledChangeCantRemoveProductWithoutPermission(self, time):
+        time.return_value = 300
         data = {"data_version": 1, "product": None, "sc_data_version": 1}
         ret = self._post("/scheduled_changes/rules/2", username="bob", data=data)
         self.assertEquals(ret.status_code, 403, ret.data)
 
-    def testUpdateRuleWithMergeError(self):
+    @mock.patch("time.time")
+    def testUpdateRuleWithMergeError(self, time):
+        time.return_value = 300
         data = {"mapping": "a", "data_version": 1}
         ret = self._post("/rules/1", data=data)
         self.assertEquals(ret.status_code, 400, ret.data)
```

Comment:
This test doesn't involve submitting a scheduled change so I'm not sure why time is being mocked. I see you've done that widely but not everywhere, so perhaps you had a reason to ?